sudo --validate failed

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

If the matter is security related, please disclose it privately via https://kubernetes.io/security/
-->


**What happened**:
when run `make test-e2e-node PARALLELISM=1 FOCUS="CriticalPod" SKIP=""`, I founded  `sudo --validate || exit 1` failed.

```
sudo：invalid option -- -
usage: sudo [-D level] -h | -K | -k | -V
usage: sudo -v [-AknS] [-D level] [-g groupname|#gid] [-p prompt] [-u user name|#uid]
usage: sudo -l[l] [-AknS] [-D level] [-g groupname|#gid] [-p prompt] [-U user name] [-u user name|#uid] [-g groupname|#gid] [command]
usage: sudo [-AbEHknPS] [-r role] [-t type] [-C fd] [-D level] [-g groupname|#gid] [-p prompt] [-u user name|#uid] [-g groupname|#gid]
            [VAR=value] [-i|-s] [<command>]
usage: sudo -e [-AknS] [-r role] [-t type] [-C fd] [-D level] [-g groupname|#gid] [-p prompt] [-u user name|#uid] file ...
make: *** [test-e2e-node] Err 1
```
**What you expected to happen**:
Test passed


**How to reproduce it (as minimally and precisely as possible)**:
Sudo version 1.8.6p7

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 1.17
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):centos
- Kernel (e.g. `uname -a`):Linux elk-141 4.9.6-200.2.el7.centos.x86_64 #1 SMP Wed Mar 1 18:39:23 CST 2017 x86_64 x86_64 x86_64 GNU/Linu
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

--- a/hack/make-rules/test-e2e-node.sh
+++ b/hack/make-rules/test-e2e-node.sh
@@ -165,7 +165,7 @@ else
     echo 'No need to refresh sudo credentials'
   else
     echo 'Updating sudo credentials'
-    sudo --validate || exit 1
+    sudo -v || exit 1
   fi
 
   # Do not use any network plugin by default. User could override the flags with

--- a/test/e2e_node/conformance/run_test.sh
+++ b/test/e2e_node/conformance/run_test.sh
@@ -34,7 +34,7 @@ elif sudo --non-interactive "$(which bash)" -c true 2> /dev/null; then
   echo 'No need to refresh sudo credentials'
 else
   echo 'Updating sudo credentials'
-  sudo --validate || exit 1
+  sudo -v || exit 1
 fi
 
 # FOCUS is ginkgo focus to select which tests to run. By default, FOCUS is

